src/main/java/seedu/foodrem/logic/commands/generalcommands/ExitCommand.java

```diff
@@ -10,14 +10,12 @@
  * Terminates the program.
  */
 public class ExitCommand extends Command {
-    private static final String MESSAGE_SUCCESS = "Exiting FoodRem as requested ...";
-
     @Override
     public CommandResult<String> execute(Model model) {
         return new CommandResult<>() {
             @Override
             public String getOutput() {
-                return MESSAGE_SUCCESS;
+                return "Exiting FoodRem as requested ...";
```

Comment:
Just curious, why the decision to refactor out the MESSAGE_SUCCESS for these commands?

Reply:
See https://github.com/AY2223S1-CS2103T-W16-2/tp/pull/286#discussion_r1005286380. Basically, it's only ever used once. In my opinion, unlike magic numbers, the purpose of these string literals are very clear. Abstracting it only hurts readability of code.